jk-barcode-demo/angular.json

```diff
@@ -32,7 +32,8 @@
             "styles": [
               "src/styles.scss"
             ],
-            "scripts": []
+            "scripts": [],
+            "preserveSymlinks": true
```

Comment:
You only need this because the library is installed using `file://`, you shouldnt when installed from npm